shared_models/shared_models.py

```diff
@@ -239,6 +241,7 @@ class PresentationExchange(BaseModel):
     ] = None
     updated_at: Optional[str] = None
     verified: Optional[bool] = None
+    thread_id: Optional[str] = None
```

Comment:
I think presentation model can also have a parent thread id?

Reply:
hmm let me double check - I didn't see parent thread id in the presentation exchange records but I'll look again

Reply:
yeah that was me - well spotted. Parent thread id is there. will add that in

Reply:
There's no parent_thread_id in V20PresExRecord nor in V10PresentationExchange. As far as I see. Can we get/derive this in a straight forward way from somewhere?

Reply:
Ah it seems it takes it from the presentation request message: https://github.com/hyperledger/aries-cloudagent-python/blob/0.7.5/aries_cloudagent/protocols/present_proof/v2_0/manager.py#L286

Reply:
ah alright thanks. That's exclusive to v2?

Reply:
Don't think so

Reply:
yeah just saw it's in both (potentially) available

Reply:
thx for the pointer